app/api/video_recordings.py

```diff
@@ -25,33 +25,36 @@ def before_get(self, args, kwargs):
                     'You need to be the event organizer to access video recordings.',
                 )
 
-            params = dict(
-                meetingID=stream.extra['response']['meetingID'],
-            )
-            channel = stream.channel
-            bbb = BigBlueButton(channel.api_url, channel.api_key)
-            result = bbb.request('getRecordings', params)
-
-            if result.data['response']['recordings']:
-                recordings = []
-                if type(result.data['response']['recordings']['recording']) is list:
-                    recordings = result.data['response']['recordings']['recording']
-                else:
-                    recordings.append(result.data['response']['recordings']['recording'])
-                for recording in recordings:
-                    get_or_create(
-                        VideoRecording,
-                        bbb_record_id=recording['recordID'],
-                        participants=recording['participants'],
-                        url=recording['playback']['format']['url'],
-                        start_time=datetime.fromtimestamp(
-                            int(int(recording['startTime']) / 1000)
-                        ),
-                        end_time=datetime.fromtimestamp(
-                            int(int(recording['endTime']) / 1000)
-                        ),
-                        video_stream=stream,
-                    )
+            if stream.extra is not None:
```

Comment:
Why??
Now I am getting **500 Internal Server Error**

![ss_1636710900](https://user-images.githubusercontent.com/42090582/141448409-349a90cf-74c0-400b-8dcb-006d79f243dd.png)

Reply:
@progmatic-99 - this will only work if you have bbb enabled, I dont think this error will happen as this has been solved in this PR

Reply:
@pc-beast Can you please check by taking PULL

Reply:
```File "/home/pc-beast/Documents/fossasia/op-ev-server/app/api/video_recordings.py", line 30, in before_get
    meetingID=stream.extra['response']['meetingID'],
KeyError: 'response'```

Reply:
I am not getting this error @pc-beast @progmatic-99 .